public/utils/index.ts

```diff
@@ -0,0 +1,6 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export * from './constants';
```

Comment:
Can't we export this directly in `constants.ts`?

Reply:
Exporting here means that the exported modules in constants are importable via `utils/` itself, rather than having to import from `utils/constants`

Main purpose of these `index.ts` files are for organizing and bubbling up exported modules so they can be imported from a higher level (`import x from ../utils/` vs. `import x from '../utils/constants`